config/helm/deployment-args.yaml

```diff
@@ -0,0 +1,50 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: capi-controller-manager
+  namespace: capi-webhook-system
+spec:
+  template:
+    spec:
+      containers:
+        - name: manager
+          args:
+            - --metrics-addr=0.0.0.0:8080
+            - --webhook-port=9443
+            - --feature-gates=MachinePool=true,ClusterResourceSet=true
+            - --watch-filter=capi
+            - --v=0
```

Comment:
Why `0` for verbosity? I feel like we usually like to keep this a little higher in case we need some visibility into reconciliation.

Reply:
ping @fiunchinho as this got merged from v1.0.x